engines/mtropolis/plugin/standard.cpp

```diff
@@ -1417,6 +1613,111 @@ const char *PanningModifier::getDefaultName() const {
 	return "Panning Modifier"; // ???
 }
 
+FadeModifier::FadeModifier() {
+}
+
+FadeModifier::~FadeModifier() {
+}
+
+bool FadeModifier::load(const PlugInModifierLoaderContext &context, const Data::Standard::FadeModifier &data) {
+	return true;
+}
+
+bool FadeModifier::respondsToEvent(const Event &evt) const {
+	return false;
+}
+
+VThreadState FadeModifier::consumeMessage(Runtime *runtime, const Common::SharedPtr<MessageProperties> &msg) {
+	return kVThreadReturn;
+}
+
+void FadeModifier::disable(Runtime *runtime) {
+}
+
+#ifdef MTROPOLIS_DEBUG_ENABLE
+void FadeModifier::debugInspect(IDebugInspectionReport *report) const {
+	Modifier::debugInspect(report);
+}
+#endif
+
+Common::SharedPtr<Modifier> FadeModifier::shallowClone() const {
+	return Common::SharedPtr<Modifier>(new FadeModifier(*this));
+}
+
+const char *FadeModifier::getDefaultName() const {
+	return "Fade Modifier";
+}
+
+OpenTitleModifier::OpenTitleModifier() {
+}
+
+OpenTitleModifier::~OpenTitleModifier() {
+}
+
+bool OpenTitleModifier::load(const PlugInModifierLoaderContext &context, const Data::Standard::OpenTitleModifier &data) {
+	return true;
+}
+
+bool OpenTitleModifier::respondsToEvent(const Event &evt) const {
+	return false;
```

Comment:
This should be stubbed out with the event from the data, since it's probably "Execute When" or similar.  `respondsToEvent` should check that event, and if `consumeMessage` doesn't handle it, then it should throw a warning, e.g.:

```
#ifdef MTROPOLIS_DEBUG_ENABLE
		if (Debugger *debugger = runtime->debugGetDebugger())
			debugger->notify(kDebugSeverityWarning, "Open Title modifier was executed, which isn't implemented yet");
#endif
```

Reply:
I have stubs for two dozen more modifiers from plugins lined up. How do I know when and when not to handle events like this?